Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM registry.fedoraproject.org/fedora:29
+
+LABEL maintainer="Factory 2 Team" \
+      description="A microservice triggered by specific message to tag a build."
+
+RUN dnf install -y \
+        python3-pyyaml \
+        python3-fedmsg \
+        python3-koji \
+        python3-requests \
+        python3-qpid-proton \
+        krb5-workstation \
+    && dnf clean all
+
+RUN sed -i '/default_ccache_name = KEYRING:persistent:%{uid}/d' /etc/krb5.conf
+
+WORKDIR /src
+
+COPY . .
```

Comment:
You should delete the `fedmsg.d` folder in `/src` after copying it or add it to your `.dockerignore` file. The reason is that fedmsg-hub will read from these config files when fedmsg-hub will be run from this directory. We ideally want the user to supply the configuration by mounting a volume, not by the config stored in the upstream source.